code/game/atoms.dm

```diff
@@ -915,6 +921,8 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 	transfer_blood = rand(2, 4)
 
 /turf/add_blood(list/blood_dna, b_color)
+	if(isnull(b_color))
+		b_color = "#A10808"
```

Comment:
just set this as the default value for the `b_color` parameter instead of doing a null check. That's what that feature is for.

Reply:
`b_color` is always transfered in proc call, placing there null sometimes, would it work with default value if we call proc with null argument?

i mean, if i make 
`/turf/add_blood(list/blood_dna, b_color="#A10808")`
and then call 
`add_blood(X, null)`
would it be `null` or `"#A10808"`?

Reply:
erghh, you're right

Reply:
I have too much faith that `null` won't just be passed as an argument 😭